htdocs/static/js/src/webfront.js

```diff
@@ -301,6 +301,33 @@ require([
 
 
     /**
+     * Import dashboard sumbit function
+     */
+    function setupImportDashboard() {
+        $('#dashboard-import form').submit(function(event) {
+            event.preventDefault();
+            var formData = new FormData($(this)[0]);
+
+            $.ajax({
+                url: $(this).attr("action"),
+                type: 'POST',
+                data: formData,
+                processData: false,
+                contentType: false,
+                success: function (data) {
+                    if (data.location) {
+                        window.location = data.location;
+                    }
+                    if (data.error) {
```

Comment:
Jeg er ikke spessielt dreven i go, så jeg tviler på at det er grunnen. Det føltes som en grei nok måte å gjøre det på. Jeg kan godt endre om det er viktig for deg